.snyk

```diff
@@ -3,49 +3,49 @@ version: v1.25.0
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
```

Comment:
Please check these changes, I think they are breaking the file

Reply:
They are actually fixing it :). It broke IaC before, and they are automatically added from the CLI `ignore` command.

Reply:
```
basti-work :: ~/workspace/snyk-ls ‹feat/HEAD-278_implement_vulnerability_count_api_client*› » snyk iac test

Snyk Infrastructure as Code

✔ Test completed.

Issues
  No vulnerable paths were found!

Test Failures

  Failed to parse JSON file
  Path: /Users/bdoetsch/workspace/snyk-ls/.vscode/launch.json
        /Users/bdoetsch/workspace/snyk-ls/.vscode/settings.json
        /Users/bdoetsch/workspace/snyk-ls/.vscode/tasks.json
        /Users/bdoetsch/workspace/snyk-ls/licenses/github.com/hashicorp/hcl/json/parser/test-fixtures/bad_input_128.json
        /Users/bdoetsch/workspace/snyk-ls/licenses/github.com/hashicorp/hcl/json/parser/test-fixtures/bad_input_tf_8110.json

-------------------------------------------------------

Test Summary

  Organization: platform_hammerhead
  Project name: snyk/snyk-ls

✔ Files without issues: 2
✗ Files with issues: 0
  Ignored issues: 4
  Total issues: 0 [ 0 critical, 0 high, 0 medium, 0 low ]

Tip: Re-run in debug mode to see more information: DEBUG=*snyk* <COMMAND>
If the issue persists contact support@snyk.io

-------------------------------------------------------

Tip

  New: Share your test results in the Snyk Web UI with the option --report
```

Reply:
Although the `null` looks weird

Reply:
Pushed a commit to remove the nulls. But the file works :)